### PR TITLE
yarGen: init at 0.23.4

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/yarGen/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/security/yarGen/default.nix:5]:
```suggestion
}:

python3.pkgs.buildPythonApplication rec {
```

File: pkgs/tools/security/yarGen/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version = "0.23.4";
+
+  src = fetchFromGitHub {
+    owner = "Neo23x0";
+    repo = "yarGen";
+    rev = version;
+    sha256 = "6PJNAeeLAyUlZcIi0g57sO1Ex6atn7JhbK9kDbNrZ6A=";
+  };
+
+  format = "other";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/yarGen/default.nix:16]:
```suggestion
```

File: pkgs/tools/security/yarGen/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "yarGen";
+  version = "0.23.4";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/yarGen/default.nix:7]:
```suggestion
  version = "0.23.4";
  format = "other";
```

File: pkgs/tools/security/yarGen/default.nix [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  };
+
+  format = "other";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    chmod +x yarGen.py
+    mv yarGen.py $out/bin
```
Comment (SuperSandro2000) [inline pkgs/tools/security/yarGen/default.nix:23]:
```suggestion
    mv yarGen.py $out/bin/yargen
```

**Comment by Ericson2314** [comment]
This + 24896293e40cecb2f22cc4db0a13c41697745049 broke staging.

**Pull request merged**